emulator/src/extensions/keymap/ext_keymap.c

```diff
@@ -52,7 +52,7 @@ const emuExtension_t keymapEmuCallback = {
     .fnInitCb        = keymapInit,
     .fnPreFrameCb    = NULL,
     .fnPostFrameCb   = NULL,
-    .fnKeyCb         = keymapKeyCb,
+    .fnKeyCb         = &keymapKeyCb,
```

Comment:
I don't think you need the `&` here. Or if you do, wouldn't you need it for `keymapInit` as well?

Reply:
Removed it.